### ISSUE TITLE
Decision on hosting at Google

### ISSUE THREAD

**Issue by alecfrancesconi** [Data4Democracy/ati-broadcastapp#1]
The AITD partner group utilizes Google (GSuite) right now and wants to host there. We need to decide if that will be sufficient; my initial resource indicates that they may have to upgrade to the Google Cloud Platform which will require an additional expense from them. This is also tied to the language decision

**Comment by alecfrancesconi** [comment]
Need to do an estimate for hosting the application on the GoogleCloud Platform

**Comment by alecfrancesconi** [comment]
Spoke to Google with Danielle yesterday. We will use the Google App Engine on their Cloud Platform. We can login with our credentials and setup the instance under their free trial. Should be able to provision everything we need from there.

**Comment by alecfrancesconi** [comment]
ATID Broadcast app environment update

The environment at Google is available under the Google App Engine portion of the Cloud Platform at this URL (https://admin.google.com/americatheindivisible.org/AdminHome?fral=1). 

Best way to access is to use the developer account (developer@americatheindivisible.org). The password has been supplied via email under separate cover. DM @alecf on slack for creds as needed. 

Once authenticated click on the App Engine Apps icon at the bottom of the admin panel. It will send you to a page with a link to the Google Developers Console. Click on that link (it opens a separate page so you might need to reset back to the developer id as it will default to your google ID if applicable.)

Once you are authorized on that page select the "ATID Broadcast App" from the menu to the left of the search bar in the top nav. Once selected you are in the app environment and the left-hand nav allows you to access various settings. Most of this is not running right now and may not need to depending on what we need. 

Directly to the right of the search bar in the top nav is an icon to "Activate Google Cloud Shell". This will activate a terminal view that allows for command-line access to the environment. 

I cloned the current GitHub repo down here but wasn't able to get it to run (we can discuss, I'm sure it's my error but I didn't spend much time trying to figure out how to get it to run). 

We discussed linking the GitHub repo to Google with Travis CI. There is a fair amount of config that is required to set this up, we can discuss if we feel this is still necessary or if we want to manually fork/clone down to Google to test and deploy as needed. This isn't a particularly complex app so the dev cycle may not require this level of complexity. We can discuss next meet.
